src/types/product.ts

```diff
@@ -0,0 +1,11 @@
+export interface Product {
+  id: string,
+  name: string,
+  imageURL: string,
+  description: string,
+  brand: string,
```

Comment:
if we are thinking of a big app to be added to your portoilio, i would like the `brand` to be an object contains brand name and logo